CHANGELOG.md

```diff
@@ -2,6 +2,76 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [5.0.0](https://github.com/assisrafael/angular-input-masks/compare/v2.5.0...v5.0.0) (2020-03-06)
```

Comment:
@evertongx thank you for your PR, but could you remove this changes? The release process cannot be mixed with feature/fixes PRs